src/components/flat-table/flat-table-checkbox/flat-table-checkbox.spec.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import TestRenderer from 'react-test-renderer';
+import FlatTableCheckbox from './flat-table-checkbox.component';
+import guid from '../../../utils/helpers/guid';
+import { Checkbox } from '../../../__experimental__/components/checkbox';
+
+jest.mock('../../../utils/helpers/guid');
+guid.mockImplementation(() => 'guid-12345');
+
+describe('FlatTableCheckbox', () => {
+  describe('the "as" prop is not passed in', () => {
+    it('renders to match the expected styling for a "td" element', () => {
+      expect(TestRenderer.create(<FlatTableCheckbox />).toJSON()).toMatchSnapshot();
```

Comment:
I don't think we should mix and match enzyme and react-test-render. You can snapshot the enzyme `mount`.

Reply:
Fair point, I'll remove enzyme